app/src/org/commcare/utils/SyncDetailCalculations.java

```diff
@@ -53,20 +58,15 @@ private static void showSyncMessage(TextView subTextView, int normalColor, int w
         } else {
             syncTimeMessage = DateUtils.formatSameDayTime(lastSyncTime, new Date().getTime(), DateFormat.DEFAULT, DateFormat.DEFAULT);
         }
+        return new Pair<>(lastSyncTime, Localization.get("home.sync.message.last", new String[]{syncTimeMessage.toString()}));
+    }
 
-        String message = "";
```

Comment:
This whole section is no longer ever relevant/used because if we're showing a sync message based upon the last sync time, that means there were no unsent forms.